src/devTools.js

```diff
@@ -90,12 +95,11 @@ export function send(action, state, options, type, instanceId) {
   }, 0);
 }
 
-export function connect(options = {}) {
+export function connect(options = {}, hostnamePromise) {
   const id = generateId(options.instanceId);
-  start(options);
   return {
     init: (state, action) => {
```

Comment:
Can we make `init` an async function and `await hostnamePromise` before passing it into `send()`?

Reply:
done